resources/keywords/suite_and_test_teardown.resource

```diff
@@ -70,3 +71,33 @@ Tear Down The Test Suite
     ...    AND    Setup DBaaSPolicy For ${operatorNamespace}
     ...    AND    Set Global Variable    ${DBaaSPolicyConfigured}    ${True}
 
+Configure DBaaS Policy Namespaces
```

Comment:
Don't understand why all of this is under teardown

Reply:
Thanks for pointing it out, I planned to refactor the resource file name and missed it out. Updated resource file name from "resources/keywords/suite_and_test_teardown.resource" to "resources/keywords/suite_and_test_setup_teardown.resource"